resumator/models.py

```diff
@@ -36,10 +37,12 @@ class Education(models.Model):
                                     default=None,
                                     verbose_name=_("Degree abbreviation"))
     start_date = models.DateField(null=True, blank=True,
-                                  verbose_name=_("start date"))
+                                  verbose_name=_("start date"),
+                                  input_formats=DATE_INPUT_FORMATS)
```

Comment:
`input_formats` doesn't work for `models.DateField` because it doesn't have it. ` forms.DateField` does, but I'm not using forms here. 

Please also look at the stackoverflow post I included in the pull request comment.